File: Competitive Programming 3/Section 1.3.3/12554 - A Special "Happy Birthday" Song!!!/Python/main.py
from sys import stdin
import math

def main():
    song=["Happy","birthday","to","you"]
    people=[]
    n=int(input())
    for line in stdin:
        people.append(line.rstrip())
    lines = math.ceil(float(n)/16)*16
    for moment in range(0,lines):
        print(people[moment%n]+": "+(song[moment%4] if moment%16!=11 else "Rujia"))
main()

File: Competitive Programming 3/Section 1.3.3/12554 - A Special "Happy Birthday" Song!!!/Java/Main.java
import java.util.Scanner;

public class Main {
    static Scanner s = new Scanner(System.in);
    static String[] song={
        "Happy",
        "birthday",
        "to",
        "you"
    },people=new String[100];
    public static void main(String[]args){
        int n =  Integer.parseInt(s.nextLine());
        for(int i = 0 ; i < n ; i++){
            people[i]=new String(s.nextLine());
        }
        int lines = (int)Math.ceil((float)n/16)*16;
        for(int moment = 0 ; moment < lines ; moment++){
            System.out.printf("%s: %s\n",people[moment%n],moment%16!=11?song[moment%4]:"Rujia");
        }
    }
}